index.html

```diff
@@ -216,6 +216,49 @@ <h3>Skills</h3>
         </ul>
       </div>
     </section>
+     <section class="feedback">
+      <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="#" method="post">
+        <input type="text" class='write' maxlength="30" placeholder="Full name" required>
+        <input type="email" class='write' placeholder="Email address" required>
+        <textarea name="msg" placeholder="Hey I'd like to connect to..." maxlength="500" required></textarea>
+        <button type="submit" class="the-button">Get in touch</button>
+      </form>
+    </section>
+   <footer>
+      <p>myemailhere@email.com</p>
+      <div class="back-right"></div>
+      <div class="social-media-links">
+        <ul>
+          <li>
+            <a href="#">
+              <img src="./assets/images/twiter.jpg" alt="Twitter">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./assets/images/linkedin.jpg" alt="Linkedin">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./assets/images/media.jpg" alt="Medium">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./assets/images/github.jpg" alt="Github">
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="./assets/images/fingers.jpg" alt="Angellist">
+            </a>
+          </li>
+        </ul>
+        <p class="divide"></p>
+      </div>
+    </footer>
```

Comment:
Please make sure your design is the same as the given Figma design. Currently, the gap between your social media is not the same as Figma. It would be better to decrease your `CSS` `gap` property to make it the same. Kindly fix it.
 Your design
![image](https://github.com/ngalamac/Ngala-Portfolio/assets/101523582/a7e1a480-31e1-47b2-b943-4c02592e43b4)

Figma design
![image](https://github.com/ngalamac/Ngala-Portfolio/assets/101523582/bb156631-1fac-4470-973d-b61b29ca2f0d)